app/src/main/java/com/grzegorzdec/tagmango/seller/map/ClientItemViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package com.grzegorzdec.tagmango.seller.map
+
+import androidx.databinding.Bindable
+import com.grzegorzdec.tagmango.BR
+import com.grzegorzdec.tagmango.model.Client
+import com.grzegorzdec.tagmango.tools.RegistryViewModel
+
+class ClientItemViewModel: RegistryViewModel() {
```

Comment:
extend ObservableViewModel and modify 
```
@get:Bindable
    var client: Client? by bindable(null)
```

Reply:
That's the thing with delegated properties and custom setters that can't be done out of the box - will change this in next iteration when I find the proper workaround

Reply:
I'll add rx decorators

Reply:
Do we need Rx when we have coroutines?